src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Generated/Models/BgpSettings.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// 
+// Code generated by Microsoft (R) AutoRest Code Generator 0.14.0.0
```

Comment:
please regenerate with 0.16.0-Nightly20160307 (you should update the version in generate.cmd file and run it)